pennylane/ops/channel.py

```diff
@@ -429,24 +428,26 @@ def _kraus_matrices(cls, *params):
         if len(operators) != len(wirelist):
             raise ValueError("The number of operators must match the number of wires")
 
-        max_q = max(wirelist)
+        nq = len(wirelist)
+
+        if (2 ** (2 * (nq)) * 8) > 1 / 4 * os.sysconf("SC_PAGE_SIZE") * os.sysconf("SC_PHYS_PAGES"):
```

Comment:
Damn, I forgot about that. I think using NumPy wouldn't make a whole lot of sense because we can only use it after creation of the Kraus matrices. A fixed number of qubits could make sense though